tests/_support/Helper/Acceptance.php

```diff
@@ -0,0 +1,10 @@
+<?php
```

Comment:
Might not need to commit these, seems like it builds them on start based on the `*.suite.yml` files.

Reply:
Not really, they're only created while creating/bootstrapping the suite, not during runtime, removing the files caused the test run to fail.